src/zbeacon.c

```diff
@@ -705,8 +706,11 @@ zbeacon_test (bool verbose)
     zactor_destroy (&node2);
     zactor_destroy (&node3);
 
-#if defined (__WINDOWS__)
-    zsys_shutdown();
+    //Unset multicast
+    zsys_set_ipv4_mcast_address (NULL);
+
+#if defined(__WINDOWS__)
```

Comment:
Remove this, and call it from before in the _test function. We should not run shutdown multiple times.